src/main/java/com/adobe/epubcheck/ops/OPSHandler30.java

```diff
@@ -679,6 +692,31 @@ else if (!isOutermostSVGAlreadyProcessed)
     }
   }
 
+  protected void processMeta(XMLElement e)
+  {
+    if (EpubConstants.HtmlNamespaceUri.equals(e.getNamespace()))
+    {
+      String name = e.getAttribute("name");
+      if ("viewport".equals(name))
+      {
+        hasViewport = true;
+        // For a fixed-layout document:
+        // check that the vieport declares both height and width
+        if (context.opfItem.isPresent() && context.opfItem.get().isFixedLayout())
+        {
+          String contentAttribute = e.getAttribute("content");
+          if (contentAttribute == null || !contentAttribute.contains("width=")
+              || !contentAttribute.contains("height="))
```

Comment:
It looks like this may fail some valid cases, like " height = 1200, width = 800".

The syntax allows spaces on either side of the equals sign: https://www.w3.org/TR/epub-33/#viewport.ebnf.assign

Reply:
Yeah, this PR basically ported the old checks as-is, good catch!
More accurate parsing of the `viewport` syntax is implemented in #1400, with newer changes not being fully taken into account but tracked as #1401.